src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {

```

Comment:
```suggestion
public interface AreaCalculator {

```
this is not area, this is interface that calculates area of a figure, so AreaCalculator